bundler/spec/support/builders.rb

```diff
@@ -423,21 +425,30 @@ def required_ruby_version=(*reqs)
     end
 
     class BundlerBuilder
-      attr_writer :required_ruby_version
+      SPEC_FILE = File.join "..", "..", "bundler.gemspec"
```

Comment:
This is causing the current CI failures `undefined method 'version=' for nil` because the path is `../../rubygems/bundler/bundler.gemspec`.